Sliding-Images-Easy.md

```diff
@@ -0,0 +1,249 @@
+# Sliding Images Easy
+
+This tutorial will walk you through building an image slider using the [jQuery](https://jquery.com/) library.
+
+[![Screenshot showing Slider on CodePen](https://i.gyazo.com/234697e8dd7838d1f75f987c8fc68adc.png)](https://codepen.io/atjonathan/pen/BKMxxq)
+
+This tutorial will have three parts:
+- [HTML](#html)
+- [SCSS](#scss)
+- [JS](#js)
+- [References](#references)
+
+## HTML
+
+We will be using [Bootstrap](http://getbootstrap.com/) for this tutorial to keep things looking good, without spending a lot of time.
+
+Our structure will be as follows:
+
+```html
+<div class="container">
+
+  <!-- The wrapper for our slider -->
+  <div class="slider">
+    <ul class="slides"><!-- Each image will be inside this unordered list --></ul>
+  </div>
+
+  <div class="buttons"><!-- Pause and play buttons will go in here --></div>
+
+</div>
+```
+
+Inside our `ul` with the class of `slides` we will have the following:
+
+> Note: We repeat the first image at the end because our javascript automatically replaces the last `li` with the first to restart the animation.
```

Comment:
That is not a good sign. It's a bad omen. May God bless us all :stuck_out_tongue_winking_eye: